Editor/EditorCore/EditShapeTool.cs

```diff
@@ -158,7 +158,7 @@ public override void OnWillBeDeactivated()
 
         public void ResetToLastSelectMode()
         {
-            if(ProBuilderToolManager.activeTool != Tool.Custom)
+            if(ProBuilderToolManager.activeTool != Tool.Custom && ProBuilderToolManager.IsAnyProBuilderContextActive())
```

Comment:
Correct me if I'm wrong, but since `ProBuilderToolManager.IsAnyProBuilderContextActive()` is defined only under the `TOOL_CONTEXTS_ENABLED` macro, shouldn't we also check for that macro when calling the method?

Reply:
@claudiaantoun The `ResetToLastSelectMode` is in an #else branch of the `#if !UNITY_2020_2_OR_NEWER` conditional which is the same conditional  as `#if !TOOL_CONTEXTS_ENABLED`

Reply:
Oh gotcha, thanks! :D